docs/pages/deployment/tls-configuration.rst

```diff
@@ -0,0 +1,99 @@
+.. _tls-configuration:
+
+TLS Configuration
+#################
+
+Connections between Nuts nodes are secured using mutual TLS (both client and server present a X.509 certificate).
+This applies to both gRPC and HTTP connections. Your TLS configuration depends mostly on where you terminate the TLS connection.
+This page describes the different layouts for TLS and how to configure them for gRPC.
+
+In all layouts your node's certificate must issued by a Certificate Authority, trusted by the other nodes in the network.
+Each layout requires ``network.certfile``, ``network.certkeyfile`` and ``network.truststorefile`` to be configured.
+
+You can also find working setups in the `end-2-end test suite <https://github.com/nuts-foundation/nuts-go-e2e-test>`_.
+
+No TLS Offloading
+*****************
+
+By default, the TLS connection is terminated on the Nuts node.
```

Comment:
Added note and reference to interface/endpoint config.

We probably need more docs regarding TLS termination on HTTPS, but out of scope for this issue.

Maybe we even need to support TLS on Echo, although that is slightly more complicated because some endpoints require TLS, some don't, and others require mTLS.

https://github.com/nuts-foundation/nuts-node/issues/1303